brave/browser-laptop
# Update "Our Partners" side bar on disabledContent.js to reflect BAT i…

…ntegration

Closes #11242 _(when #11231 is merged to `master`)_

Auditors: @cezaraugusto

Test Plan:
1. Run `npm run unittest`
1. Open about:preferences#payments
2. Make sure 'Brave Bitcoin Wallets' is replaced with 'Brave BAT Wallets'
3. Make sure the uphold company logo is displayed

<img width="1021" alt="screenshot 2017-10-03 15 42 07" src="https://user-images.githubusercontent.com/3362943/31112898-75da1bac-a851-11e7-8b40-6fe3d67af765.png">


Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
